src/index.css

```diff
@@ -10,7 +10,7 @@
   h4,
   h5,
   h6 {
-    @apply font-heading;
+    @apply font-bold;
```

Comment:
```suggestion
    @apply font-heading font-bold;
```
should not remove `font-heading`, many `<h>` titles are in `montserrat`

also if we are going to abstract `font-bold` here, be sure to remove `font-bold` in all `<h>` that have this className

Reply:
Done with the requested change @ap-justin , apologies , i wasn't aware clicking the commit suggestion auto-resolved the comment

Reply:
The changes were already committed at the time of this comment being resolved, so all was according to protocol.
@ap-justin try reloading the PR files page next time, it happens that GH doesn't show the most recent changes.

Reply:
> @ap-justin try reloading the PR files page next time, it happens that GH doesn't show the most recent changes.

@misicnenad, what do you mean? I've re-requested changes 20 hours ago, 
<img width="876" alt="image" src="https://user-images.githubusercontent.com/89639563/226886899-122c1cbf-b3a5-4845-98f6-9489cc36d28b.png">

while the changes regarding this were just made 6 hours ago
<img width="922" alt="image" src="https://user-images.githubusercontent.com/89639563/226886979-48b9c2de-f9df-452b-9f6b-6b84de610fea.png">

Reply:
@ap-justin you are correct in pointing that out, i was working on these changes as well but i made the mistake of accepting "commit suggestion" which prematurely resolved the ticket, no harm done , i hv done the changes now